server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/HistogramAggregationBuilder.java

```diff
@@ -194,11 +194,17 @@ public HistogramAggregationBuilder offset(double offset) {
 
     /** Get the current minimum bound that is set on this builder. */
     public double minBound() {
+        if (extendedBounds.getMin() == null) {
+            return Double.NEGATIVE_INFINITY;
```

Comment:
I think this should be other way around: `POSITIVE_INFINITY` for min and `NEGATIVE_INFiNITY` for max. Otherwise specifying null extended bounds would mean we want bounds to extend bounds to the entire range. This should be also consistent with the behaviour which was before this issues was introduced in #60556. 

Please also note that extendedBounds themselves can be null, so I would suggest using [`DoubleBounds#getEffectiveMin`](https://github.com/elastic/elasticsearch/blob/bfc62a419e7cd1f7f8bc0a3026a2f6ecc361fd45/server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/DoubleBounds.java#L140) instead.